src/main/java/org/sagebionetworks/web/client/widget/entity/editor/EntityListConfigEditor.java

```diff
@@ -88,16 +89,26 @@ public void refresh() {
 
 	@Override
 	public void onAddRecord() {
-		entityFinder.configure(true, new SelectedHandler<Reference>() {
-			@Override
-			public void onSelected(Reference selected) {
-				entityFinder.hide();
-				EntityGroupRecord record = createRecord(selected.getTargetId(), selected.getTargetVersionNumber(), null);
-				entityListWidget.addRecord(record);
-				view.setButtonToolbarVisible(true);
-			}
-		});
-		entityFinder.show();
+		entityFinderBuilder
+				.setModalTitle("Insert Item List")
+				.setInitialScope(EntityFinderScope.CURRENT_PROJECT)
+				.setInitialContainer(EntityFinder.InitialContainer.PROJECT)
+				.setVisibleTypesInList(EntityFilter.ALL)
+				.setSelectableTypes(EntityFilter.ALL)
+				.setHelpMarkdown("Search or Browse Synapse to find Projects, Folders or Files and insert them as a list into this Wiki page")
+				.setPromptCopy("Find items in Synapse to insert them as a list")
+				.setMultiSelect(true)
+				.setShowVersions(true)
+				.setSelectedMultiHandler(((selected, entityFinder) -> {
+					entityFinder.hide();
+					for (Reference ref : selected) {
+						EntityGroupRecord record = createRecord(ref.getTargetId(), ref.getTargetVersionNumber(), null);
+						entityListWidget.addRecord(record);
+					}
+					view.setButtonToolbarVisible(true);
+				}))
```

Comment:
This widget now supports selecting multiple entities

Reply:
Thanks for enhancing this while you were here